third_party/terraform/website/docs/r/google_service_account_key.html.markdown

```diff
@@ -25,28 +25,6 @@ resource "google_service_account_key" "mykey" {
 }
 ```
 
-## Example Usage, save key in Kubernetes secret
```

Comment:
I removed this example as Workload Identity is a better way of achieving this.

Reply:
I agree with you, but I think we want to keep the example.  If you want to add, like, "deprecated - see workload identity for the recommended path", that would be okay!

Reply:
fixed